src/families/location_scale.jl

```diff
@@ -30,8 +30,8 @@ This is necessary to guarantee stable convergence.
 function MvLocationScale(
     location::AbstractVector{T},
     scale::AbstractMatrix{T},
-    dist::ContinuousDistribution;
-    scale_eps::T=sqrt(eps(T)),
+    dist::ContinuousUnivariateDistribution;
+    scale_eps::T=eps(T)^(1//4),
```

Comment:
I'm curious what provoked the `1//4` power.

Reply:
This was an arbitrary choice because `1//2` seem to result in unstable behavior in some cases. I think I will just fix it to 1e-4 or 1e-3 regardless of the type. I don't think we expect posteriors with a variance of 1e-8 to exist in practice.